packages/components/src/Toast/__snapshots__/Toast.spec.tsx.snap

```diff
@@ -69,29 +69,6 @@ exports[`<Toast /> Dismissable story renders snapshot 1`] = `
       You've got toast!
     </p>
   </div>
-  <button
-    class="close button variantLink colorSuccess"
-    type="button"
-  >
```

Comment:
side effect of the actions addOn is `onDismiss` isn't in the prepared story for these snapshots. I think it's fine since we have Percy testing here anyways